tools/AutoTuner/src/autotuner/distributed.py

```diff
@@ -1001,7 +1011,7 @@ def sweep():
             local_dir=LOCAL_DIR,
             resume=args.resume,
             stop={"training_iteration": args.iterations},
-            resources_per_trial={"cpu": args.resources_per_trial},
+            resources_per_trial={"cpu": os.cpu_count() / args.jobs},
```

Comment:
What would happen if the machine has 16 CPUs and the user asks for more than 16 jobs, e.g., 32,64...? Should there be a check/warning/error message here?